test/setup/index.ts

```diff
@@ -35,7 +35,10 @@ export function snykMonitorNamespace(): string {
 }
 
 export async function removeMonitor(): Promise<void> {
-  await dumpLogs();
+  // Credentials may have expired on certain platforms (OpenShift 4), try to regenerate them.
+  await platforms[testPlatform].config().catch(() => undefined);
+  await dumpLogs().catch(() => undefined);
+
```

Comment:
```suggestion
  try {
    await platforms[testPlatform].config();
    await dumpLogs();
  } catch {
  
  }
```

Reply:
I would still like it to try dumping the logs if it cannot do platforms.config()! Best-effort for all of these 😬